### PR TITLE
revert PR #553 related change

### PR DESCRIPTION
**Pull request by songleo**
https://github.com/open-cluster-management/backlog/issues/12393
https://github.com/open-cluster-management/backlog/issues/13302

Signed-off-by: Song Song Li <ssli@redhat.com>

### CODE REVIEW

File: pkg/controller/multiclusterobservability/multiclusterobservability_controller.go [hunk 1]
```diff
@@ -385,6 +385,12 @@ func (r *ReconcileMultiClusterObservability) UpdateStatus(
 	return nil, nil
 }
 
+// labelsForMultiClusterMonitoring returns the labels for selecting the resources
+// belonging to the given MultiClusterObservability CR name.
+func labelsForMultiClusterMonitoring(name string) map[string]string {
+	return map[string]string{"observability.open-cluster-management.io/name": name}
+}
+
```
Comment (clyang82) [inline pkg/controller/multiclusterobservability/multiclusterobservability_controller.go:393]: What is it for? I did not find where to call it?

Comment (songleo) [reply pkg/controller/multiclusterobservability/multiclusterobservability_controller.go:393]: this is dead code, I will remove it. thanks.

**Comment by songleo** [comment]
need to retrigger e2e job to publish image when this PR merge: https://travis-ci.com/github/open-cluster-management/observability-e2e-test/builds/228698680

**Pull request merged**